kernel-configs/qcom-imsdk.config

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Don't think we need copyright statement for the config fragments, but fine to have it.

Reply:
Dropped from this new config, and also added a commit to drop it from the other config